extensions/hibernate-orm/runtime/src/main/java/io/quarkus/hibernate/orm/runtime/tenant/HibernateMultiTenantConnectionProvider.java

```diff
@@ -107,20 +105,18 @@ private static ConnectionProvider resolveConnectionProvider(String persistenceUn
      * @return Current tenant resolver.
      */
     private static InstanceHandle<TenantResolver> tenantResolver(String persistenceUnitName) {
-        InstanceHandle<TenantResolver> resolverInstance;
-        if (PersistenceUnitUtil.isDefaultPersistenceUnit(persistenceUnitName)) {
-            resolverInstance = Arc.container().instance(TenantResolver.class, Default.Literal.INSTANCE);
-        } else {
-            resolverInstance = Arc.container().instance(TenantResolver.class,
-                    new PersistenceUnitLiteral(persistenceUnitName));
```

Comment:
This is the old annotation. The `legacySingleExtensionInstanceForPersistenceUnit` method despite its name supports both the new annotation and the old one if the new one is not found.